index_release.go

```diff
@@ -0,0 +1,23 @@
+package versionbundle
+
+import (
+	"time"
+)
+
+const indexReleaseTimestampFormat = "2006-01-02T15:04:05.00Z"
+
+type IndexRelease struct {
+	Active      bool        `yaml:"active"`
+	Authorities []Authority `yaml:"authorities"`
+	Date        time.Time   `yaml:"date"`
+	Version     string      `yaml:"version"`
+}
+
+// TODO define and implement validation rules
```

Comment:
What should be the rules? 

- Release versions should not be duplicated.
- Versions should be valid semver versions. 
- Date should not be empty. 
- Authority endpoint should not be empty.

Reply:
@tuommaki since I am not there tomorrow and the version bundle fixes are prio 1 for the team feel free to take over here and start implementing some validation. Doesn't have to be complete nor perfect in the end but then we can go over with the other tasks from the list and iterate towards the actual fix.

Reply:
Do we want to enforce semver format to `Version` string? Would we have use cases for `2.3.1-patchX` or possibly `2.3.1+kvm` or anything along the line?

Reply:
I thought so. There are several libs doing this so we should just be able to use these.

Reply:
So only the semver validation is missing currently?

Reply:
And the renaming. We will come back soon.